.gitignore

```
@@ -1 +1,2 @@
 games.log
+coverage
```

Essa pasta não tá sendo ignorada pelo arquivo, se você ver na sua PR todos os arquivos referentes ao coverage foram adicionados nos seus commits.

```suggestion
coverage/
```